### PR TITLE
Replace assignment with equality check in example

### PR DESCRIPTION
**Pull request by wurde**
First time reading the ActionCable README, but I believe this ought to be an equality check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,13 +51,14 @@ module ApplicationCable
...
     protected
-      def find_verified_user
-        if current_user = User.find(cookies.signed[:user_id])
-          current_user
-        else
-          reject_unauthorized_connection
-        end
+
+    def find_verified_user
+      if current_user == User.find(cookies.signed[:user_id])
```
Comment (seuros) [inline README.md]: ```User.find_by_id(cookies.signed[:user_id])``` since ```#find``` will raise if the user don't exists.

**Comment by lsylvester** [comment]
I think that it should be an assignment, otherwise the `current_user` that is returned on the next line will be nil, and then `self.current_user` will be set to nil.

**Comment by wurde** [comment]
But Rubocop didn't like assignments in conditions. I think you're right though and I'll take a closer look.

**Comment by wurde** [comment]
Confirmed, thank you.

**Pull request closed without merging**